c_src/h3.c

```diff
@@ -640,7 +866,10 @@ static ErlNifFunc nif_funcs[] =
      {"k_ring", 2, erl_k_ring, 0},
      {"k_ring_distances", 2, erl_k_ring_distances, 0},
      {"max_k_ring_size", 1, erl_max_k_ring_size, 0},
-     {"indices_are_neighbors", 2, erl_indices_are_neighbors, 0}};
+     {"indices_are_neighbors", 2, erl_indices_are_neighbors, 0},
+     {"polyfill", 2, erl_polyfill, 0},
```

Comment:
Could you make polyfill be marked for the dirty scheduler? You're right in that polyfill can take a long time, so if we put this function on a dirty scheduler at least we won't affect the rest of the VM.  Set ERL_NIF_DIRTY_JOB_CPU_BOUND on the CPU intensive ErlNifFunc flag entries (where it says 0 right now)

Reply:
Yup, I had that on there temporarily and can add the flag back.